main.cpp

```diff
@@ -184,6 +200,21 @@ struct DoubleType
 
     operator double() const { return *a; }
 
+    DoubleType& apply( std::function<DoubleType&( double& )> func )
+    {
+        if( func != nullptr )
+            func( *a );
+        return *this;
+    }
+
+    using funcPtr = void(*)( double& );
```

Comment:
`using` directives follow the same rules as Type naming.
- [ ] Capitalize your alias.